src/main/scala/mesosphere/marathon/api/TaskKiller.scala

```diff
@@ -43,22 +47,28 @@ class TaskKiller @Inject() (
           val activeInstances = foundInstances.filter(_.isActive)
 
           if (wipe) {
-            val instancesAreTerminal: Future[Done] = killService.watchForKilledInstances(activeInstances)
+            val foundInstanceIds = Set(foundInstances.map(_.instanceId): _*)
```

Comment:
Just wonder, why not 
```suggestion
            val foundInstanceIds = foundInstances.map(_.instanceId).toSet?
```